Documentation/cad-models.rst

```diff
@@ -4,10 +4,20 @@ CAD models
 The following help pages describe how to build a structural model with
 Gocad.
 
-A simple tutorial
------------------
 
-See :doc:`generating-a-cad-model-using-gocad-basic-tutorial`.
+SimModeler CAD workflow
+-----------------------
+
+Since September 2019, SimModeler features powerfull tools for processing discrete data (geometry in the form of meshes), which allow building structural models without the need to rely on additionnal CAD software.
```

Comment:
powerful~~l~~ addition~~n~~al

Reply:
ok you are right, I checked also this file with Grammarly now. Thx.